app/models/room.rb

```diff
@@ -5,6 +5,8 @@ class Room < ApplicationRecord
   validates :name, presence: true
   after_initialize :generate_public_id
 
+  belongs_to :user, optional: true
```

Comment:
do we want it to be optional [right now] though? If we don't delete users, might be simpler to assume the user is always there, so we could do `Created by Tomek` without worrying that user might be not set